libflux/go/libflux/analyze.go

```diff
@@ -1,6 +1,7 @@
 package libflux
 
-// #include "influxdata/flux.h"
+// #cgo CFLAGS: -I${SRCDIR}/../../include/influxdata
+// #include <flux.h>
```

Comment:
For some reason, `/`s inside of `include` don't work, but `/`s inside of `cgo CFLAGS` _do_ work.

Reply:
Empirically windows accepts forward slashes as equivalent to backslashes 'most of the time'. Perhaps header parsing is more pedantic?